sdk/azidentity/CHANGELOG.md

```diff
@@ -1,15 +1,10 @@
 # Release History
 
-## 1.4.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.4.0-beta.2 (2023-07-14)
 
 ### Other Changes
 * `DefaultAzureCredentialOptions.TenantID` applies to workload identity authentication
+* Upgraded dependencies
```

Comment:
Entry for the switch to mutex?

Reply:
Just for historical interest? It has no observable effect

Reply:
Good point.  The behavior didn't change.